### PR TITLE
Update VCPKG baseline and improve documentation

### PR DESCRIPTION
**Pull request by Meakk**

### CODE REVIEW

File: doc/dev/GETTING_STARTED.md [hunk 1]
```diff
@@ -86,9 +86,14 @@ mkdir build
 
  * Open git bash
 
+Check the VCPKG baseline in `vcpkg.json` at the root of f3d source directory.  
+It should be `c9140a3b500812ad3206317885860d9553b93f13` (please let us know if it's not)
```
Comment (mwestphal) [inline doc/dev/GETTING_STARTED.md:90]: What does it mean to "check the VCPKG baseline" ?

Keep in mind this guide is for complete beginners. If VCPKG doesnt work out of the box, we may consider modifying the guide to build VTK manually.

Comment (Meakk) [reply doc/dev/GETTING_STARTED.md:90]: > modifying the guide to build VTK manually

Definitely not. I don't want to write a guide for every single dependency.

> Check the VCPKG baseline in `vcpkg.json` at the root of f3d source directory.

I'll be more clear, but that's something a beginner can check easily.

Comment (Meakk) [reply doc/dev/GETTING_STARTED.md]: Let me know if it's better now.

Comment (mwestphal) [reply doc/dev/GETTING_STARTED.md]: On windows, VTK is the only needed dependency afaik

Comment (Meakk) [reply doc/dev/GETTING_STARTED.md]: And plenty of other optional dependencies. I still think VCPKG is way more beginner-friendly.

Comment (mwestphal) [reply doc/dev/GETTING_STARTED.md]: I agree

File: doc/dev/GETTING_STARTED.md [hunk 2]
```diff
@@ -86,9 +86,14 @@ mkdir build
 
  * Open git bash
 
+Check the value of `builtin-baseline` in the `vcpkg.json` file at the root of f3d source directory.  
+Make sure it's `c9140a3b500812ad3206317885860d9553b93f13` (please let us know if it's not)
```
Comment (mwestphal) [inline doc/dev/GETTING_STARTED.md:90]: I don't think that comment is important? As long as we ensure the hash in the guide is the same as the one in the JSON file unit will work, right ?

Comment (Meakk) [reply doc/dev/GETTING_STARTED.md:90]: We may forgot right? There's nothing in the CI that ensure it's the case

Comment (Meakk) [reply doc/dev/GETTING_STARTED.md:90]: Can we come up with a bash command to read a value in a json file and check if the value is found in the documentation file?

Comment (mwestphal) [reply doc/dev/GETTING_STARTED.md:90]: To me its similar to forgetting that archlinux dependencies are optional for vtk, we just need to put good info in the guide. If we change it in one location, we need to change it in the other location. Its just doc.

Comment (Meakk) [reply doc/dev/GETTING_STARTED.md:90]: Fair enough. I'll remove the comment, but you have been warned ^^

Comment (mwestphal) [reply doc/dev/GETTING_STARTED.md:90]: move this forward ?

Comment (Meakk) [reply doc/dev/GETTING_STARTED.md]: done

**Review by mwestphal — APPROVED** [review]

**Pull request merged**